po/fr.po

```diff
@@ -0,0 +1,196 @@
+# Copyright (C) 2019 THE com.github.torikulhabib.nino'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the com.github.torikulhabib.nino package.
+# NathanBnm, 2019.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: com.github.torikulhabib.nino\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2018-11-17 10:36+0700\n"
+"PO-Revision-Date: 2019-01-19 21:07+0100\n"
+"Last-Translator: NathanBnm\n"
+"Language-Team: Français\n"
+"Language: fr\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+
+#: data/com.github.torikulhabib.nino.appdata.xml.in:7
+#: data/com.github.torikulhabib.nino.desktop.in.in:3
+msgid "Nino"
+msgstr "Nino"
+
+#: data/com.github.torikulhabib.nino.appdata.xml.in:8
+#: data/com.github.torikulhabib.nino.desktop.in.in:4
+msgid "A network monitor widget"
+msgstr "Un widget de moniteur réseau"
+
+#: data/com.github.torikulhabib.nino.desktop.in.in:5
+msgid "Network Monitor"
+msgstr "Moniteur réseau"
+
+#: data/com.github.torikulhabib.nino.appdata.xml.in:10
+msgid "An internet speed monitor apps."
+msgstr "Une application de surveillance de la vitesse d'Internet."
+
+#: data/com.github.torikulhabib.nino.appdata.xml.in:11
+msgid "Features:"
+msgstr "Fonctionnalités :"
+
+#: data/com.github.torikulhabib.nino.appdata.xml.in:13
+msgid "Small window application features"
+msgstr "Toutes les fonctionnalités de l'application dans une petite fenêtre"
+
+#: data/com.github.torikulhabib.nino.appdata.xml.in:14
+msgid "Alert when no internet connection is connected."
+msgstr "Soyez alerté lorsqu'aucune connexion Internet n'est établie."
+
+#: data/com.github.torikulhabib.nino.appdata.xml.in:15
+msgid "Many color theme choices available."
+msgstr "Plusieurs choix de couleurs de thème disponibles."
+
+#: data/com.github.torikulhabib.nino.appdata.xml.in:16
+msgid "Lock position on the desktop."
+msgstr "Verrouillez la position sur le bureau."
+
+#: data/com.github.torikulhabib.nino.appdata.xml.in:17
+msgid "Save the position of the widget."
+msgstr "Enregistrez la position du widget"
+
+#: data/com.github.torikulhabib.nino.appdata.xml.in:18
+msgid "The choice is always above the screen or below."
+msgstr "Le choix est toujours au-dessus de l'écran ou en dessous."
+
+#: src/MainWindow.vala:79
+msgid "Desktop"
+msgstr "Bureau"
+
+#: src/MainWindow.vala:94
+msgid "Window"
+msgstr "Fenêtre"
+
+#: src/MainWindow.vala:118
+msgid "Mini Window"
+msgstr "Petite fenêtre"
+
+#: src/MainWindow.vala:136 src/Widgets/Preferences.vala:204
+#: src/MiniWindow.vala:89
+msgid "Close"
+msgstr "Fermer"
+
+#: src/MiniWindow.vala:71
+msgid "Position"
+msgstr "Position"
+
+#: src/MainWindow.vala:222 src/MiniWindow.vala:187
+msgid "Network Is Not Available"
+msgstr "Réseau indisponible"
+
+#: src/MainWindow.vala:280
+msgid "Connect to the Internet to Monitoring Network."
+msgstr "Connectez-vous à Internet pour surveiller le réseau."
+
+#: src/MainWindow.vala:288
+msgid "Network Settings"
```

Comment:
For some reason "…" lacks from the source string, maybe you deleted it by mistake?

Reply:
For a reference:

![screenshot from 2019-03-05 12-28-17](https://user-images.githubusercontent.com/26003928/53779061-315bc500-3f42-11e9-8300-cd101bc5a368.png)

Reply:
I don't remember I have deleted any symbol. I will fix this too! Thank you for comment.